src/elements/core/styles/core.scss

```
@@ -196,6 +196,14 @@ sbb-lead-container {
   }
 }
 
+// TODO: Move back to the sbb-message components when the global css refactoring happens
```

Nitpicky 100%, but see my other open PR :D
```suggestion
// TODO: Move back to the sbb-message components when the global CSS refactoring happens
```